src/industry_gui.cpp

```diff
@@ -1256,6 +1256,14 @@ class IndustryDirectoryWindow : public Window {
 	StringID cargo_filter_texts[NUM_CARGO + 3]; ///< Texts for filter_cargo, terminated by INVALID_STRING_ID
 	byte produced_cargo_filter_criteria;        ///< Selected produced cargo filter index
 	byte accepted_cargo_filter_criteria;        ///< Selected accepted cargo filter index
+	static CargoID produced_cargo_filter;
```

Comment:
Why make this variable static? Something to do with maintaining filter if you close/open the window? Is that worthwhile?

Reply:
I don't know. I copied over from @glx22